fairlearn/metrics_plots/plot_disparities_in_metric.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mtick
+from fairlearn.metrics import MetricFrame
+
+
+def plot_disparities_in_metric(metric, y_true, y_pred, sensitive_features, show_plot=True):
+    """Plot disparities in a metric between groups defined by sensitive features.
+
+    This method visualizes the metric values for all groups as identified
+    by sensitive features.
+
+    Parameters
+    ----------
+    metric : callable
+        A metric function that takes arguments `y_true` and `y_pred`
+    y_true : array-like
+        The list of true values
+    y_pred : array-like
+        The list of predicted values
+    sensitive_features : array-like
+        the sensitive attributes
+
+    """
+    # compute
+    metric_frame = MetricFrame(metric, y_true, y_pred,
+                               sensitive_features=sensitive_features)
+
+    # chart text for localization
+    metric_text = metric.__name__.replace('_', ' ')
+    title_text = f'Disparity in {metric_text}'
+    ylabel_text = 'By group'
+    overall_performance_text = 'Overall'
+    diff_text = 'min-max difference'
+    ratio_text = 'min-max ratio'
+    # chart styles
+    figsize = (12, 4)
```

Comment:
this should also be an optional input parameter, or done via accepting `ax`

Reply:
Agreed. The caller can adjust that if they want. I'm removing this line.